cli/pcluster/config/pcluster_config.py

```diff
@@ -296,6 +307,41 @@ def __init_section_from_file(self, section_definition, config_parser, section_la
         except configparser.NoSectionError as e:
             self.error("Section '[{0}]' not found in the config file.".format(e.section))
 
+    def __refresh(self):
```

Comment:
Sorry I don't get this point. The method which is meant to be invoked publicly is refresh(). This one is just a subset of the operations done during a PcusterConfig refresh() and it is private by purpose, because invoking it directly would not be safe and could lead to incorrect config settings.

Reply:
OK, maybe your comment was referred to the previous version. The new logic should fit your requirements. Let me know pls.

Reply:
yes correct, thanks.